app/infrastructure/persistence/repositories/oauth_provider_config_repo.py

```
@@ -51,15 +79,107 @@ def _serialize_for_audit(self, obj: OAuthProviderConfig) -> dict[str, Any]:
             select(OAuthProviderConfig).where(
                 OAuthProviderConfig.id == config_id,
                 OAuthProviderConfig.tenant_id == tenant_id,
+                OAuthProviderConfig.deleted_at.is_(None),
             )
         )
         return result.scalar_one_or_none()
 
+    async def list_configs(
+        self,
+        tenant_id: str,
+        include_inactive: bool = False,
```

Ruff (0.15.0)

[error] Boolean-typed positional argument in function definition

(FBT001)